source/developer/aarch64.rst

```diff
@@ -0,0 +1,25 @@
+``aarch64`` builds
+==================
+
+.. datechanged:: 2023-11-18
+   Added this section as initial linux-aarch64 builds are starting
+
+While we do not yet have builds for ``osx-arm64`` (see
+:ref:`platform-nomenclature-faq`), we are starting to roll out
+``linux-aarch64`` builds. These run on CircleCI, which offers the
+infrastructure for this (in contrast to GitHub Actions).
```

Comment:
This comment will be obsolete soon - https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/

Reply:
I am not suggesting to wait for Github Actions to add support for ARM! They postponed this several times. I'm just saying that the sentence will (probably) become obsolete soon, so it might be either removed or changed to:
```suggestion
infrastructure for this (in contrast to GitHub Actions, which does not support Linux ARM64 at the moment).
```